person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+
+  attr_accessor :age, :name
+
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] @sam2if Great work implementing the changes requested, however, you haven't implemented the last request of removing the id from the parameters as id is not passed as a parameter but generated randomly
```suggestion
     def initialize(age, name = "Unknown", parent_permission : true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

![image](https://github.com/sam2if/school_library_people/assets/61408860/db19a5e8-fc3b-4426-a97b-204570190443)